.github/workflows/upload.yml

```
@@ -4,25 +4,34 @@ on:
     types: [published]
 
 jobs:
+  determine_runner:
+    name: Change to Large Runner
+    uses: ./.github/workflows/determine-workflow-runner.yml
+    with:
+      default_runner: ubuntu-latest
+      force_large_runner: ${{ inputs.use_large_runner }}
```

This workflow doesn't have any inputs. `force_large_runner` should always be `true`.

```suggestion
      force_large_runner: true
```